app/assets/javascripts/discourse/app/lib/flag-targets/flag.js

```diff
@@ -13,7 +13,7 @@ export default class Flag {
     // an instance of ActionSummary
     const postAction = this.postActionFor(flagModal);
     flagModal.appEvents.trigger(
-      this.flagCreatedEvent,
+      this.flagCreatedEvent(),
```

Comment:
it would be better instead to convert all those methods (title, customSubmitLabel, submitLabel, flagCreatedEvent) into getters. (that would require updating e.g. d-post-voting though)